specification/translation/Azure.AI.DocumentTranslation/main.tsp

```diff
@@ -39,4 +39,7 @@ namespace DocumentTranslation;
 enum Versions {
   @useDependency(Azure.Core.Versions.v1_0_Preview_1)
   v2024_05_01: "2024-05-01",
+
+  @useDependency(Azure.Core.Versions.v1_0_Preview_1)
+  v2024_11_15_preview: "2024-11-15-preview",
```

Comment:
Update to the correct version: https://machinetranslation.visualstudio.com/MachineTranslation/_git/mt-api-frontend?path=/private/MTBatchService/Microsoft.MT.Batch.Frontend/Controllers/V1/ControllerConstants.cs&version=GBmaster&_a=contents

Reply:
Updated